game_template/.gitignore

```diff
@@ -33,7 +33,6 @@
 .fvm/
 
 # Web related
```

Comment:
probably should delete this line as well

Reply:
Fixed.